index.html

```diff
@@ -3932,6 +3932,52 @@ <h3>Transformations</h3>
         </p>
       </section>
 
+      <section>
+        <h3>Protected Information</h3>
+
+        <p>
+The data that is protected by any [=data integrity proof=] is the
+[=transformation|transformed data=]. [=transformation|Transformed data=]
+is generated by a [=transformation algorithm=] that is specified by a
+particular [=cryptosuite=]. This protection mechanism is in contrast to some
+more traditional digital signature mechanisms that do not perform any sort of
+[=transformation=] on the input data. The benefits of [=transformation=] are
+detailed in Section [[[#transformations]]].
+        </p>
+
+        <p>
+For example, [=cryptosuites=] such as
+<a data-cite="?VC-DI-ECDSA#ecdsa-jcs-2019">ecdsa-jcs-2019</a> and
+<a data-cite="?VC-DI-EDDSA#eddsa-jcs-2022">eddsa-jcs-2022</a> use the
+[[[?RFC8785]]] to [=transformation|transform=] the data to canonicalized JSON,
+which is then cryptographically hashed and digitally signed. One benefit of this
+approach is that adding or removing formatting characters that do not impact the
+meaning of the information being signed, such as spaces, tabs, and newlines,
+does not invalidate the digital signature. More traditional digital signature
+mechanisms do not have this capability.
+        </p>
+
+        <p>
+Other [=cryptosuites=] such as
+<a data-cite="?VC-DI-ECDSA#ecdsa-rdfc-2019">ecdsa-rdfc-2019</a> and
+<a data-cite="?VC-DI-EDDSA#eddsa-rdfc-2022">eddsa-rdfc-2022</a> use
+[[[?RDF-CANON]]] to [=transformation|transform=] the data to canonicalized
+N-Quads [[?N-QUADS]], which is then cryptographically hashed and digitally
+signed. One benefit of this approach is that the cryptographic signature is
+portable to a variety of different syntaxes, such as JSON, YAML, and CBOR,
+without invalidating the digital signature. More traditional digital signature
+mechanisms do not have this capability.
+        </p>
+
+        <p>
+Implementers and developers are urged to not presume or trust information
+that contains a [=data integrity proof=] unless the proof has been [=verified=]
+and the verified data is provided in a return value from a software library
+that has confirmed that all data returned has been successfully protected.
```

Comment:
There might instead be some presumption which was inadvertently omitted from "presume ... information", and which restoration would make this sentence make more sense.
```suggestion
Implementers and developers are urged to not trust information
that contains a [=data integrity proof=] unless the proof has been [=verified=]
and the verified data is provided in a return value from a software library
that has confirmed that all data returned has been successfully protected.
```